.github/workflows/self-push-amd.yml

```diff
@@ -76,11 +81,7 @@ jobs:
         # `CI_SHA_WORKFLOW_RUN`: The commit SHA that triggers this workflow by `workflow_run` event
         # `CI_SHA`: The non-empty commit SHA from the above two (one and only one of them is empty)
         run: |
-          CI_BRANCH_PUSH=${{ github.event.ref }}
```

Comment:
We should move the corresponding comments about the different CI values to the new place

Reply:
Thanks, will do before merge